README.md

```diff
@@ -51,22 +51,30 @@ There are different ways of creating providers inside Pulumi but for this projec
 
 ## How to Use
 
-1. Create you Pulumi project file by copying the example (`cp Pulumi.example.yaml Pulumi.yaml`) and setting name, runtime (keep as it is) and description.
+Before starting make sure you login with your Pulumi CLI. 
 
-3. For CI/CD environments, you need to add same environment variables to your secrets in your system. In the case of `GitHub Actions`, you can use `Secrets` as described [here](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets). The mapping is as following:
+1. Create you Pulumi project file by copying the example (`cp Pulumi.example.yaml Pulumi.yaml`) and setting `name`, `runtime` (keep as it is) and `description`.
```

Comment:
It is a old error but as you are already changing it, would you change "you" to "your" please?